cli/lib/fetch.js

```diff
@@ -1,4 +1,4 @@
-"use strict";
+'use strict';
```

Comment:
Prettier

Reply:
I think #183 takes care of this, but hopefully #149 can prevent in the future :slightly_smiling_face: